Plugins/Public/playercntl_plugin/HyperJump.cpp

```diff
@@ -1903,25 +1580,39 @@ namespace HyperJump
 
 			pub::Player::SendNNMessage(iClientID, pub::GetNicknameId("nnv_jumpdrive_blind_jump_warning"));
 
-			vector<string> systems;
-			struct Universe::ISystem *sysinfo = Universe::GetFirstSystem();
-			while (sysinfo)
-			{
-				systems.push_back(sysinfo->nickname);
-				sysinfo = Universe::GetNextSystem();
-			}
-			bool isBannedSystem = true;
-			while(isBannedSystem)
+			if (!JumpSystemListEnabled)
 			{
+				vector<string> systems;
+				struct Universe::ISystem *sysinfo = Universe::GetFirstSystem();
+				while (sysinfo)
+				{
+					systems.push_back(sysinfo->nickname);
+					sysinfo = Universe::GetNextSystem();
+				}
```

Comment:
This block could be:

```suggestion
				for (struct Universe::ISystem *sysinfo = Universe::GetFirstSystem(); sysinfo; sysinfo = Universe::GetNextSystem())
				{
					systems.push_back(sysinfo->nickname);
				}
```